auto_submit/test/service/config_test.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:auto_submit/foundation/providers.dart';
+import 'package:auto_submit/service/config.dart';
+import 'package:auto_submit/service/secrets.dart';
+import 'package:neat_cache/cache_provider.dart';
+import 'package:neat_cache/neat_cache.dart';
+import 'package:test/test.dart';
+
+/// Number of entries allowed in [Cache].
+const int kCacheSize = 1024;
+void main() {
+  group('Config', () {
+    late CacheProvider cacheProvider;
+    late Config config;
+    const int kCacheSize = 1024;
+    final HttpProvider httpProvider = Providers.freshHttpClient;
```

Comment:
Use `MockClient` instead. Unit tests should not use http

https://github.com/dart-lang/http/blob/master/lib/testing.dart

Reply:
Thank you Casey! I studied some example codes and changed our unit test to use mock client here.